docs/GroupDefs.hpp

```diff
@@ -58,6 +58,16 @@
  * with the Jacobian and inverse Jacobian of the maps.
  */
 
+/*!
+ * \defgroup CoordMapsTimeDependentGroup  Time-dependent Coordinate Maps
```

Comment:
Please make it `Coordinate Maps, Time-dependent` so the list appears alphabetical :-)